### PR TITLE
[IMP] inventory: update wh/loc doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] targeting the `warehouses_locations` doc in Odoo's Inventory app documentation, specific to 17.0.

Here's what I did:
- Rewrote/updated copy and sections for accuracy specific to 17.0
- Replaced screenshots/added new screenshots with 17.0 UI
- Added admonitions and expanded doc to be more robust

[Project Task Link](https://www.odoo.com/web#id=3904009&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by brse-odoo** [comment]
@jero-odoo This PR is ready for content review when you have a chance, thanks!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 1]
```diff
@@ -8,88 +8,110 @@ Terminology
 Warehouse
 ---------
 
-In Odoo, a *warehouse* is the actual building/place in which a company's items are stocked. Multiple
-warehouses can be set up in Odoo and the user can create moves between warehouses.
+In Odoo *Inventory*, a *warehouse* is a physical building or space where items are stored. By
+default, each Odoo database has one warehouse already pre-configured, with the address set as
+the company's address.
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:13]:
```suggestion
default, each Odoo database has one warehouse already pre-configured, with the address set as the
company's address.
```
early line break on 12 (double check with make review, but moving "the" from 13 to 12 should fix it)

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 2]
```diff
@@ -8,88 +8,110 @@ Terminology
...
+A location is only part of *one* warehouse, and can **not** be linked to multiple warehouses. As
+many locations can be configured as needed under one warehouse.
 
-There are three types of locations:
+There are three types of locations in Odoo *Inventory*:
 
-- The **Physical Locations** are internal locations that are part of the warehouses that the
-  company owns. They can be the loading and unloading areas of the warehouse, a shelf, a department,
-  etc.
+- **Physical Locations**: internal locations within a single warehouses owned by the company. This
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:31]:
```suggestion
- **Physical Locations**: internal locations within a single warehouse owned by the company. This
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 3]
```diff
@@ -8,88 +8,110 @@ Terminology
...
+  can include shipping and loading docks, aisles, shelves, storage areas, and more.
 
-- The **Partner Locations** are spaces within a customer and/or vendor's warehouse. They work the
-  same way as physical locations, with the only difference being that they are not owned by the
-  user's company.
+- **Partner Locations**: external locations, such as designated spaces in a customer and/or vendor's
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:34]:
```suggestion
- **Partner Locations**: external locations, such as designated spaces in a customer or vendor's
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 4]
```diff
@@ -8,88 +8,110 @@ Terminology
...
 
 Configuration
 =============
 
-To activate locations, go to :menuselection:`Configuration --> Settings` and enable
-:guilabel:`Storage Locations`. Then, click :guilabel:`Save`.
+For companies with one warehouse, no additional settings are required to activate before editing the
+warehouse.
+
+For companies with multiple warehouses, and to use create locations within a warehouse, the *Storage
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:53]:
```suggestion
For companies with multiple warehouses, and to create locations within a warehouse, the *Storage
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 5]
```diff
@@ -8,88 +8,110 @@ Terminology
...
+Locations* feature must be enabled.
+
+To do so, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and scroll down
+to the :guilabel:`Warehouse` section. Tick the checkbox next to :guilabel:`Storage Locations` to
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:57]:
```suggestion
to the :guilabel:`Warehouse` section. Tick the :guilabel:`Storage Locations` checkbox to
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 6]
```diff
@@ -8,88 +8,110 @@ Terminology
...
 
-To create a warehouse, go to :menuselection:`Configuration --> Warehouse Management --> Warehouses`
-and click on :guilabel:`Create`.
+To create a new warehouse, navigate to :menuselection:`Inventory app --> Configuration -->
+Warehouses`, and click :guilabel:`New`. Doing so opens a blank warehouse creation form.
 
-Then, fill out a :guilabel:`Warehouse Name` and a :guilabel:`Short Name`. The short name is five
-characters maximum.
+Begin by assigning a :guilabel:`Warehouse Name`, and a :guilabel:`Short Name`. The :guilabel:`Short
+Name` is limited to a **five** character maximum.
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:76]:
```suggestion
Name` field is limited to a **five** character maximum.
```

**Review by jero-odoo — APPROVED** [review]
hey @brse-odoo looks good, just a few suggestions, let me know if you have any questions. Thanks!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 1 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:13]: Nice catch - Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 2 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:31]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 3 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:34]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 4 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:53]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 5 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:57]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 6 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:76]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
@odoo/us-doc-review This PR is ready for final content review, thank you!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 7]
```diff
@@ -8,88 +8,110 @@ Terminology
...
+Once the warehouse is created, navigate back to the :menuselection:`Inventory` dashboard. New
+operations for the newly created warehouse are automatically generated there.
 
-.. image:: warehouses_locations/new-transfer-types.png
+For more on creating new warehouses, see :doc:`create_a_second_warehouse`.
+
+.. image:: warehouses_locations/warehouses-locations-new-wh-operations.png
    :align: center
-   :alt: Inventory app dashboard displaying new transfer types for the recently created warehouse.
+   :alt: Inventory app dashboard displaying new transfers for the new  warehouse.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:96]: Remove extra space
```suggestion
   :alt: Inventory app dashboard displaying new transfers for the new warehouse.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 8]
```diff
@@ -8,88 +8,110 @@ Terminology
...
+many locations can be configured as needed under one warehouse.
 
-There are three types of locations:
+There are three types of locations in Odoo *Inventory*:
 
-- The **Physical Locations** are internal locations that are part of the warehouses that the
-  company owns. They can be the loading and unloading areas of the warehouse, a shelf, a department,
-  etc.
+- **Physical Locations**: internal locations within a single warehouse owned by the company. This
+  can include shipping and loading docks, aisles, shelves, storage areas, and more.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:33]: No need for an empty line between bullets of the same level :)
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo, your improvements to this doc look good to me 👍 Just a couple small fixed and should be good to go. Approving and delegating.
...
@robodoo delegate=brse-odoo

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 7 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:96]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst (hunk 8 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:33]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
@robodoo r+

**Pull request closed without merging**